presto-tests/pom.xml

```diff
@@ -185,6 +180,13 @@
         </dependency>
 
         <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-resource-group-managers</artifactId>
+            <scope>test</scope>
+            <type>test-jar</type>
```

Comment:
It's better to put `type` before `scope` since it's part of the artifact identifier (Maven coordinates), whereas `scope` is part of the import.